client/src/components/employer-form/BeforeYouBegin.js

```diff
@@ -76,11 +76,7 @@ const BeforeYouBegin = () => {
             &nbsp;that may be interested in participating in the Health Career Access Program.
           </Typography>
           <Typography variant='body1' paragraph>
-            <b>
-              The Employer Expression of Interest (EOI) remains open for submissions, however intake
-              is not currently active. Any EOIs received may be considered for potential future
-              phases of the program.
-            </b>
+            <b>The Employer Expression of Interest (EOI) is now open for submissions</b>
```

Comment:
Tiny nitpick - missing a period at the end here :)

Reply:
Thank you.

Reply:
Fixed